ipn/ipnlocal/filesharingstatus_noop.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
I don't think any of this windows registry code should live in the "ipnlocal" package.

We already have "util/systemd" for systemd notifications. How about "util/osshare" for OS-specific sharing?

Reply:
Sure, makes sense.